requirements.txt

```diff
@@ -4,17 +4,18 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-amqp==1.4.9               # via kombu
+-e git+https://github.com/xuhang57/billiard.git#egg=billiard
```

Comment:
yes, there are some local modifications for this and we don't want to put it in the CCI-MOC yet since things might change and we don't want to maintain it also 👍  Thanks for pointing it out.